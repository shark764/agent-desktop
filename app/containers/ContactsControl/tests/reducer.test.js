import contactsControlReducer from '../reducer';

describe('contactsControlReducer', () => {
  it('returns the initial state', () => {
    expect(contactsControlReducer(undefined, {})).toMatchSnapshot();
  });
});
