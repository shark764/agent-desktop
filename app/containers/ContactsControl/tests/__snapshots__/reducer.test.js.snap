// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactsControl Reducer returns the initial state 1`] = `
Object {
  "contactForm": Object {},
  "formErrors": Object {},
  "formIsDirty": false,
  "formIsValid": false,
  "selectedIndexes": Object {},
  "showCancelDialog": false,
  "showConfirmDialog": false,
  "showErrors": Object {},
  "unusedFields": Object {},
}
`;
